01_01_Basic_Rules.txt

```diff
@@ -425,6 +421,19 @@ class has a Proficiency with a given armor type then they can purchase or wear
 that armor type in combat. If your character doesn't have that proficiency, then
 they can't.
 
+===== Assigning Skills to a Character =====
+
+	Characters gain points in a wide variety of skills to represent their experience
+and expertise in different fields and activities. Every time your character levels up,
+they gain skill points equal to double your Skill Point Gain. Assign each skill point to
+a skill as you see fit; see CharacterCreation/06_Skills for details about skills and for
+a list of commonly-used skills in Compound X. At level 1, your character will have two
+times their Skill Point Gain to assign at will. You may gain a maximum of 60 skill
+points in a single skill. In addition, each time you level up, you may gain a maximum of
+6 points in any one skill by using your Skill Point Gain. Skill gained from Feats, Class

```

Comment:
'using your Skill Point Gain' sounds strange to me, and I'm not certain that it's necessary to the sentence. Nonblocking

Reply:
Good point